packages/lesswrong/lib/voting/votingSystems.tsx

```diff
@@ -357,9 +358,12 @@ export function getVotingSystems(): VotingSystem[] {
   return Object.keys(votingSystems).map(k => votingSystems[k]!);
 }
 
-export async function getVotingSystemNameForDocument(document: VoteableType, context: ResolverContext): Promise<string> {
+export async function getVotingSystemNameForDocument(document: VoteableType, collectionName: VoteableCollectionName, context: ResolverContext): Promise<string> {
+  if (collectionName === "MultiDocuments" || collectionName === "Tags") {
+    return "reactionsAndLikes";
+  }
   if ((document as DbComment).tagId) {
-    return "twoAxis";
+    return "namesAttachedReactions";
```

Comment:
FYI: we will forum-gate this to `eaEmojis` for the EA forum, but you should sanity check that that's correct.  (This is for comments on tag discussions.)